content/en/docs/01/_index.md

```diff
@@ -21,12 +21,12 @@ As OpenShift is based on Kubernetes, its concepts also apply to OpenShift which
 ## Core concepts
 
 {{% onlyWhenNot openshift %}}
-With the open source software Kubernetes, you get a platform to deploy your application in a container and operate it at the same time.
-Therefore, Kubernetes is also called a _Container Platform_ or the term _Container-as-a-Service_ (CaaS) is used.
+With the open-source software Kubernetes, you get a platform to deploy your application in a container and operate it at the same time.
```

Comment:
I don't agree. See e.g. here: https://opensource.com/resources/what-open-source

Reply:
It appears that open source is missing a hyphen. Consider adding the hyphen(s).

A hyphen is used to create compound words. Often, the two words joined by the hyphen act as an **adjective** that describes a noun in the sentence.

Reply:
It seems both variants ("open source" and "open-source") are used equally falsely or correctly.